pallets/ema-oracle/src/types.rs

```diff
@@ -35,33 +35,50 @@ pub type Balance = u128;
 /// A price is a tuple of two `u128`s representing the numerator and denominator of a rational number.
 pub type Price = EmaPrice;
 
-/// A type representing data produced by a trade or liquidity event. Timestamped to the block where
-/// it was created.
+/// A type representing data produced by a trade or liquidity event.
+/// Includes the block number where it was created/updated.
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 #[derive(RuntimeDebug, Encode, Decode, Clone, PartialEq, Eq, Default, TypeInfo, MaxEncodedLen)]
 pub struct OracleEntry<BlockNumber> {
 	pub price: Price,
 	pub volume: Volume<Balance>,
 	pub liquidity: Liquidity<Balance>,
-	pub timestamp: BlockNumber,
+	pub updated_at: BlockNumber,
+}
+
+impl<BlockNumber> OracleEntry<BlockNumber> {
+	/// Construct a new `OracleEntry`. Will invert the price to determine `inverted_price`.
```

Comment:
I couldn't find any mention of `inverted_price` in this file. Could you clarify it to me please?
We can also place this method next to other methods defined for the struct.

Reply:
oops, copy pasta error, good catch 👍